packages/domains-table/src/utils/get-domain-type-text.tsx

```diff
@@ -1,9 +1,15 @@
 import { type as domainTypes, domainInfoContext } from './constants';
-import { ResponseDomain } from './types';
 import type { __ as __type } from '@wordpress/i18n';
 
+type PartialDomainInfo = {
+	tldMaintenanceEndTime?: number;
+	type: string;
+	isSubdomain: boolean;
+	isPremium: boolean;
+};
+
 export function getDomainTypeText(
-	domain: ResponseDomain,
+	domain: PartialDomainInfo,
```

Comment:
This change is not needed anymore, as I changed new code.
However, it might be worth to use `Pick` here.